templates/search/search_rfm/view.sql

```diff
@@ -0,0 +1,45 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.search.search_rfm` AS
+SELECT
+  STRUCT(
+    udf.bits_to_days_seen_int(days_seen_bytes) AS frequency,
+    udf.bits_to_first_seen_int(days_seen_bytes) AS T,
+    udf.bits_to_first_seen_int(days_seen_bytes)
+        - udf.bits_to_last_seen_int(days_seen_bytes) AS recency
```

Comment:
Should we factor out this logic to a UDF like `udf.days_seen_bytes_to_rfm(days_seen_bytes)`?